node/node.go

```diff
@@ -424,7 +424,7 @@ func ensureLease(ctx context.Context, leases v1beta1.LeaseInterface, lease *coor
 		case errors.IsNotFound(err):
 			log.G(ctx).WithError(err).Info("Node lease not supported")
 			return nil, err
-		case errors.IsAlreadyExists(err):
+		case errors.IsAlreadyExists(err), errors.IsConflict(err):
 			if err := leases.Delete(ctx, lease.Name, metav1.DeleteOptions{}); err != nil && !errors.IsNotFound(err) {
```

Comment:
If there's a conflict, likely there should be a warning log.

Reply:
I think in either case (already exists, or conflict) there should be a warning.